### PR TITLE
Move filters to standardfilters.rb

### PR DESCRIPTION
**Pull request by bnelso71**
+ Add tests for string, array, mixed, and number
+ Add activesupport/inflector (until we can find a better solution for
more complex transformations)

### CODE REVIEW

**Comment by bnelso71** [comment]
If we think it would be better to separate our filters from the standard one, we can also extract them to an `AvantFilters` class, or something like that.

I didn't move the date filter, because it conflicts with the date filter in the standard filters. If we need different date logic, we might need to change the name or alter the existing one.

File: lib/liquid.rb [hunk 1]
```diff
@@ -73,11 +73,7 @@ module Liquid
 require 'liquid/utils'
 require 'liquid/tokenizer'
 require 'liquid/parse_context'
-require 'liquid/array'
-require 'liquid/date'
-require 'liquid/mixed'
-require 'liquid/number'
-require 'liquid/string'
+# require 'liquid/date'
```
Comment (Kuraiou) [inline lib/liquid.rb]: unnecessary comment :)

File: lib/liquid/standardfilters.rb [hunk 2]
```diff
@@ -376,9 +377,96 @@ def default(input, default_value = ''.freeze)
...
+      input.titleize
+    end
+
+    def pluralize(num, singular, plural)
+      num == 1 ? singular : plural
+    end
 
     private
 
+    # From Rails ordinalize source
```
Comment (Kuraiou) [inline lib/liquid/standardfilters.rb]: 👍

**Comment by Kuraiou** [comment]
re-:tiger:

**Pull request merged**